.github/workflows/main.yml

```
@@ -0,0 +1,64 @@
+name: julia 1.0/1.3
+on:
+  - push
+  - pull_request
+    
+jobs:
+  testMKLjl:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.0'
+          - '1.3'
+        os:
+          - ubuntu-latest
+          - macOS-latest
+#           - windows-latest
+        arch:
+          - x64
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
```

```suggestion
      - uses: julia-actions/setup-julia@latest
```
This make sure that we receive latest Julia fixes.
https://github.com/julia-actions/setup-julia/releases